deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -610,6 +653,23 @@ namespace WEX
 #define TAEF_REGISTER_FIXTURE_METHOD(methodName, methodType)
 #define TAEF_REGISTER_TEST_METHOD(methodName)
 #else
+
+#if defined (__clang__)
+#define TAEF_REGISTER_FIXTURE_METHOD(methodName, methodType) \
+    struct TAEF_FixtureInvoker \
+    { \
+        static HRESULT __cdecl TAEF_Invoke(void* pTestClass) \
+        { \
+            /* TAEF_TestClassType was typedef'd in the TEST_CLASS macro so we can safely cast from \
+               void* to TAEF_TestClassType* and invoke the correct derived type's member functions */ \
+            return WEX::SafeInvoke(WEX::FixtureInvokeFunctor<TAEF_TestClassType>(*reinterpret_cast<TAEF_TestClassType*>(pTestClass), &TAEF_TestClassType::methodName)); \
```

Comment:
>&TAEF_TestClassType::methodName [](start = 133, length = 31)

wouldn't this work for TAEF, too?  Can/should we - instead of forking these entire macros - only fork the portions that need to be different?